Fix readme pr (#1008)
NethermindEth/warp: readme.md

Warp brings Solidity to Starknet, making it possible to transpile Ethereum
smart contracts to Starknet Cairo Contracts.

:warning: **Note**: Cairo 1 support is being developed at this [branch](https://github.com/NethermindEth/warp/tree/cairo-1.0).

## Quickstart

Docker compose provides a ready to use environment featuring warp and devnet.

> **Note:**

> > Executing Warp using Docker works only for x86 architecture, ARM architectures (such as Apple's M1) will be supported soon.

Reviewer: There is a second note like that below

Reviewer: It should also be updated

Warp brings Solidity to Starknet, making it possible to transpile Ethereum
smart contracts to Starknet Cairo Contracts.

> :warning: **Note**: Support for Cairo 0.10 has been reduced in order for our team to focus on upgrading Warp to be compatible with the upcoming release of Cairo 1. This upgrade will provide much more readable transpiled contracts with improved performance. All new development is being done at this [branch](https://github.com/NethermindEth/warp/tree/cairo-1.0).

Reviewer: Support wasn't reduced :sweat_smile: Let's make it concise:
```suggestion
:warning: **Note**: Cairo 1 support is being developed at this [branch](https://github.com/NethermindEth/warp/tree/cairo-1.0).
```

## Installation :gear:

> You can install warp in one of two methods, read more about it [here](installation.md)

Reviewer 2: ```suggestion
To install Warp follow the installation instructions [here](installation.md)
```

Reviewer 2: Add another line afterwards

```
To build Warp for development see the [contributing guidelines](link)
```

## Installation :gear:

> If you want to use Warp, learn how to install it [here](installation.md). If you want to contribute to its development, [here](contributing.md) are the instructions to set it up.

Reviewer: `you can learn how to install it` sounds better for me

## Contributing

> You can read about contributing, first steps, devlopment tips and testing for contributors [here](contributing.md)

Reviewer: development *
